bftengine/src/bftengine/PrimitiveTypes.hpp

```diff
@@ -26,6 +26,7 @@ namespace bftEngine
 	{
 
 
+                // TODO (AJS) - Why are SeqNum and ViewNum int64_t and not uint64?
 		typedef int64_t SeqNum;
```

Comment:
Internally we want to use SeqNum in signed expressions – e.g.,  s1-s2>0. It is possible to use uint64_t (to earn a bit), but we will have to check all expressions with SeqNum.

Reply:
Got it. Thanks.